### PR TITLE
DROOLS-5274: Remove spreadsheet file type picker

### PR DESCRIPTION
**Pull request by jomarko**
Ensemble with:
- https://github.com/kiegroup/kie-wb-common/pull/3285

The spreadsheet file type picker was removed because:
- the extension of file can be derived from file upload widget
- file upload widget doesn't allow submit other than XLS/XLSX files

For more details see https://issues.redhat.com/browse/DROOLS-5274

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
failed tests:
```
org.kie.server.integrationtests.jbpm.UserTaskServiceIntegrationTest.testCompleteAlreadyCompletedUserTask[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40008/kie-server-services/services/rest/server'}]
org.kie.server.integrationtests.jbpm.UserTaskServiceIntegrationTest.testCompleteAlreadyCompletedUserTask[1: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40008/kie-server-services/services/rest/server'}]
org.kie.server.integrationtests.jbpm.UserTaskServiceIntegrationTest.testCompleteAlreadyCompletedUserTask[2: XSTREAM KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40008/kie-server-services/services/rest/server'}]
```

reruning build to see if they are random

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java [hunk 1]
```diff
@@ -237,6 +250,7 @@ public void testSuccessMultiByteProjectNameAndFileName() {
                                                         "default://" + encode( "ああ" ) + "/src/main/resources" );
 
         when( pkg.getPackageMainResourcesPath() ).thenReturn( resourcesPath );
+        when(uploadWidget.getFilenameSelectedToUpload()).thenReturn(fileName + ".xls");
```
Comment (dupliaka) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java:253]: Small suggestion would you mind adding xlsx test as well?

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java (hunk 1 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java:291]: Done

**Comment by jomarko** [comment]
@dupliaka thank you for good spot in missing tests. Added.

I decided to apply also kiegroup code style so there is a lot of just white space changes now. I recommend to use `w=1` parameter during review as:

```
https://github.com/kiegroup/drools-wb/pull/1355/files?w=1
```

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
@kiegroup/gatekeepers may I ask for merge together with https://github.com/kiegroup/kie-wb-common/pull/3285

Downstream failed on OOME tracked as [BXMSPROD-793](https://issues.redhat.com/browse/BXMSPROD-793)

The OOME was logged after `kie-wb-distributions` execution that is last in our https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/script/repository-list.txt

I grepped console log:
- ' Failures: ' - 8120 matches
- ' Failures: 0' - 8120 matches

I think it is fine to merge.

**Pull request merged**